### PR TITLE
Subscription page display price tables

### PR DESCRIPTION
**Pull request by PopDaph**
The code from PlansTables has mostly been copied from @Duncid's PR here: https://github.com/dust-tt/dust/pull/2305
🙇🏻‍♀️ 

IN my screenshot the price is in dollar but I updated in the code.

### Big layout
Capture d’écran 2023-11-02 à 16 58 26

### Small layout
Capture d’écran 2023-11-02 à 17 01 50

### CODE REVIEW

**Review by fontanierh — DISMISSED** [review]
LGTM

**Comment by PopDaph** [comment]
Actually if we want to display the button do downgrade we need to handle stopping the subscription > I'm on it

**Review by spolu — DISMISSED** [review]

File: front/pages/w/[wId]/subscription/index.tsx [hunk 1]
```diff
@@ -1,8 +1,16 @@
-import { Button, Chip, Page, PageHeader, ShapesIcon } from "@dust-tt/sparkle";
+import {
+  Button,
+  Chip,
+  ExternalLinkIcon,
+  Page,
+  PageHeader,
```
Comment (Duncid) [inline front/pages/w/[wId]/subscription/index.tsx:6]: This is deprecated, should use Page.Header instead

File: front/components/PlansTables.tsx [hunk 2]
```diff
@@ -0,0 +1,286 @@
...
+    return (
+      <div
+        className={classNames(
+          "mx-0 sm:mx-24",
+          "w-full max-w-md px-2 py-16 sm:px-0",
+          className
+        )}
+      >
+        <Tab.Group>
+          <Tab.List className="flex space-x-1 rounded-full border border-slate-600/40 bg-slate-900/40 p-1 backdrop-blur">
```
Comment (Duncid) [inline front/components/PlansTables.tsx:183]: Need to create a "light" version of this.
You can copy the style of the BarHeader.

**Comment by PopDaph** [comment]
@Duncid we're in a hurry so I will merge as is but I will handle your feedback in a PR tomorrow 🙏🏻

**Pull request merged**